Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2678,6 +2678,12 @@ namespace OpenInfraPlatform
 				* \note
 				*/
 				template <>
+				carve::geom::vector<3> getPointOnCurve(const EXPRESSReference<typename IfcEntityTypesT::IfcThirdOrderPolynomialSpiral>& thirdOrderPolynomial,
+					const typename IfcEntityTypesT::IfcParameterValue& parameter) const noexcept(false)
+				{
+					return getPointOnCurve(thirdOrderPolynomial, parameter * this->UnitConvert()->getLengthInMeterFactor());
+				}
+				template <>

```

Comment:
I can't select the lines below in the review, so:

Line 2692/2693 - the second function parameter should be `double` and not `IfcParameterValue`.

Reply:
Resolved in [bf6802c](https://github.com/tumcms/Open-Infra-Platform/pull/566/commits/bf6802c2e85df55ec447ff0f2c7f663b4dd121ba)